src/patternfly/components/Table/examples/table-compound-expansion-example.hbs

```diff
@@ -26,19 +26,19 @@
       {{#> table-th table-th--data-label="Repository Name"}}
         <a href="#">siemur/test-space</a>
       {{/table-th}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--modifier="pf-m-expanded" table-td--data-label="Branches"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--modifier="pf-m-expanded pf-m-no-padding" table-td--data-label="Branches"}}
```

Comment:
did you mean to add `pf-m-no-padding` to the compound expansion toggle cells? Doesn't look like they have any padding by default.

Reply:
yes that was a mistake thanks for catching that!